docs/releases/readme.md

```diff
@@ -145,6 +151,24 @@ _Outcome_: **Customers can install/update via WPORG; WPORG plugin page is up to
 -   Edit the milestone and add the current date as the due date (this basically is used for easy reference of when the milestone was completed).
 -   Close the milestone.
 -   If you didn't release a patch release, create a milestone for the next minor release.
+-   Close any epics that are completed as of this release and remove any unfinished issues in that from the epic.
+
+#### Update wiki with any changes
+
+- By now the testing notes for the release should have a document in `/docs/releases` along with updating the [table of contents there](../testing/releases/README.md).
+- Also update the [wiki sidebar](https://github.com/woocommerce/woocommerce-gutenberg-products-block/wiki/_Sidebar/_edit) with the new links for the release testing notes.
```

Comment:
Is release notes are in the codebase, why don't we include a link to the release directory on the wiki (or sidebar) and remove this extra step? Link to /releases/ and you can see the directory structure and navigate to where you want.

Reply:
Ya good point, I'll make that change.